torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -3104,6 +3145,15 @@ Tensor* TensorExprKernel::bindInput(const torch::jit::Value* input) {
             input->debugName() + "' are unknown";
         throw malformed_input(msg);
       }
+      if (isContiguous(input)) {
+        Placeholder inBuffer(
+            "t" + input_name_map_[input],
+            ToDtype(static_cast<ScalarType>(*tt->scalarType())),
+            toExprHandles(*tt->sizes().concrete_sizes()));
+        bufs_.emplace(input, inBuffer.data());
+        bufferArgs_.emplace_back(inBuffer);
+        break;
```

Comment:
We probably should do `result = new Tensor(inBuffer, nullptr)` here before we break. I think we generally expect returned tensors to be not-null (but they might have a nullptr stmt signifying that there is no stmt needed to compute them)